### PR TITLE
Manifest v2

### PR DESCRIPTION
**Pull request by gabriel-okteto**

### CODE REVIEW

File: .stignore [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.git
```
Comment (pchico83) [inline .stignore:1]: For these samples, I would keep the `.stignore` file simple. 
I think we only need:
```
.git
node_modules
```

File: okteto.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: hello-world
```
Comment (pchico83) [inline okteto.yml]: Let's remove `name`, it's optional

File: okteto.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+build:
+  hello-world:
+    context: .
+    dockerfile: Dockerfile
```
Comment (pchico83) [inline okteto.yml]: we can remove `dockerfile`, that's the default value

File: okteto.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+name: hello-world
+build:
+  hello-world:
+    context: .
```
Comment (pchico83) [inline okteto.yml:4]: let's add the `image` field as we did here:
https://github.com/okteto/go-getting-started/pull/10/files#diff-dd8070292ce806a326a3d0543b136ba3b5f7930f40d33475780462fdcece2b70R3

I would use a different name, for example `okteto.dev/node-hello-world:1.0.0`.

This is the image we should use in the k8s deployment manifest, like here:
https://github.com/okteto/go-getting-started/pull/10/files#diff-9aa5c3faef5c8a780db30dc0dd41b946e5732af5677ee2fc17d3db774fe1a57bR16

File: okteto.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    dockerfile: Dockerfile
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto/dev:latest
```
Comment (pchico83) [inline okteto.yml]: remove `image`. for nodejs applications, we reuse the same image as the original deployment

File: okteto.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    autocreate: true
```
Comment (pchico83) [inline okteto.yml]: remove `autocreate`, it should attach to the running deployment

File: okteto.yml [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    command: npm i && npm run start
```
Comment (pchico83) [inline okteto.yml]: `command: bash`

File: okteto.yml [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    workdir: /usr/src/app
```
Comment (pchico83) [inline okteto.yml]: this shouldn't be needed

File: okteto.yml [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+    sync:
+      - .:/usr/src/app
+    forward:
+      - 3000:3000
```
Comment (pchico83) [inline okteto.yml]: remove port `3000`. It's better to access the app though the https endpoint

File: okteto.yml (hunk 5 above)
Comment (pchico83) [reply okteto.yml]: You didn't remove the `image` field

File: okteto.yml [hunk 10]
```diff
@@ -0,0 +1,14 @@
+build:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
```
Comment (pchico83) [inline okteto.yml:3]: You need to update the deployment in `k8s.yml` to use this image

File: .stignore (hunk 1 above)
Comment (pchico83) [inline .stignore:1]: Just keep `.git` and `node_modules` in this file to keep it simple

File: okteto.yml (hunk 8 above)
Comment (gabriel-okteto) [reply okteto.yml]: those are removed, command I left with bash

File: okteto.yml [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+    context: .
+deploy:
+  - kubectl apply -f k8s.yml
+dev:
+  hello-world:
+    image: okteto.dev/node-hello-world:1.0.0
```
Comment (pchico83) [inline okteto.yml]: @gabriel-okteto could you remove the `image` field here? It's not needed. If it's not specified, it uses the same image than the k8s deployment

Comment (gabriel-okteto) [reply okteto.yml]: ?  we went back and forth on this I think, I am removing it again

**Review by pchico83 — APPROVED** [review]

**Pull request merged**